README.md

```diff
@@ -17,6 +17,17 @@ If you have a local copy of the ISO, you can point to it with the `--iso` parame
 
 You can also add additional kernel command line arguments (e.g. `"console=ttyS0"`) to the generated configuration files using the `--extra-args` parameter.
 
+## Usage
+
+Copy the files generated under the interim folder `/tmp/tmpxxx/ubuntu-installer/`
+to your tftp root folder for netboot, for example `/var/lib/tftpboot`:
+
```

Comment:
Yes, thanks. I was hoping that you could actually `mv /tmp/tmpxxx/ubuntu-installer /var/lib/tftpboot`, and then use `ubuntu-installer/grubnet<arch>.efi` as the filename. i.e., use the `ubuntu-installer` namespace. But when I tried this, `GRUB` wasn't able to find the grub.cfg file :( I thought this used to work with `d-i`, but maybe the `grubnet<arch>.efi` it provides has a different built-in config?)

Reply:
Using `ubuntu-installer` namespace is making more sense so I tried this as well. And right, `grubnet<arch>.efi` always looks for the root folder of tftp and ubuntu-installer namespace does not work : (